README.md

```diff
@@ -6,52 +6,64 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
 
-[`@lumigo/tracer`](https://) is Lumigo's distributed-tracing and performance monitoring agent for Node.js.
+This is [`@lumigo/tracer`](https://), Lumigo's Node.js agent for distributed tracing and performance monitoring.
 
-Supported Runtimes: 8.10, 10.x, 12.x
+Supported NodeJS runtimes: 8.10, 10.x, 12.x
 
  
 ## Usage 
 
+The `@lumigo/tracer` package allows you to pursue automated metric gathering through Lambda Layers, automated metric gathering and instrumentation through the Serverless framework, or manual metric creation and implementation.
+
 ### With Lambda Layers:
-* Use the latest ARN version [from these tables](https://github.com/lumigo-io/lumigo-node/blob/master/layers)
+
+* When configuring your Lambda functions, include the appropriate Lambda Layer ARN [from these tables](https://github.com/lumigo-io/lumigo-node/blob/master/layers)
```

Comment:
Need to state that using layers is optional and in case you decide to use this aws capability then the list of layers is available from ...

Reply:
Added a note to the end of the Lambda Layer section!